src/components/hero/rotating-hero-banner.js

```
@@ -91,260 +69,291 @@ const RotatingHeroBanner = ({ disableAutoRotate = false }) => {
-                <p dangerouslySetInnerHTML={{ __html: currentContent.description }} />
-                <div className="hero-banner-read-more">Read Article <span>→</span></div>
+          )}
+
+          <div className="hero-banner-content">
+            <div className="hero-banner-text">
+              <span className="hero-banner-kicker">
+                {currentContent.kicker}
+              </span>
+              <h2>{currentContent.title}</h2>
+              <p
+                dangerouslySetInnerHTML={{ __html: currentContent.description }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)